core/src/main/java/org/secondbase/core/moduleconnection/WebConsole.java

```diff
@@ -0,0 +1,26 @@
+package org.secondbase.core.moduleconnection;
+
+import java.io.IOException;
+
+/**
+ * A web server to host SecondBase modules.
+ */
+public interface WebConsole {
+    /**
+     * Start the webconsole server.
+     * @throws IOException If there's trouble starting it.
```

Comment:
Remove dot, I don't think this counts as a sentence with @throws. Also start with lowercase.